cabal-install/Distribution/Client/Init/Defaults.hs

```diff
@@ -21,11 +21,11 @@ import Distribution.ModuleName
   ( ModuleName )  -- And for the Text instance
 import qualified Distribution.ModuleName as ModuleName
   ( fromString )
-import Distribution.Version
-  ( Version, mkVersion )
+import Distribution.CabalSpecVersion
+  ( CabalSpecVersion (..))
 
-defaultCabalVersion :: Version
-defaultCabalVersion = mkVersion [1,10]
+defaultCabalVersion :: CabalSpecVersion
+defaultCabalVersion = CabalSpecV1_10
```

Comment:
Optional: can you change this to `CabalSpecV2_4` to match other places? I missed the default value in the prompt and only defaulted it under non-interactive init when unspecified.

Reply:
I'll change non-interactive to actually use values from `Defaults`

Reply:
SG.